elfi/methods/results.py

```diff
@@ -203,6 +204,36 @@ def sample_means_array(self):
         """
         return np.array(list(self.sample_means.values()))
 
+    def save_samples(self, name=None, type_='csv'):
+        """Save samples in csv or json file formats.
+
+        Parameters
+        ----------
+        name : file name to be saved
+        type_ : file format to be saved
+
+        """
+        import json
+        import csv
+
+        if not name:
```

Comment:
Though few will probably give `name=0`, this is still a dangerous style that has bit us a couple of times already.